.github/workflows/build.yml

```diff
@@ -6,6 +6,9 @@ name: build browser extension
 
 on:
   push:
+    branches:
```

Comment:
Might wanna run it on all branches since it's cheap. Then you can write:

```
on: [push, pull_request, workflow_dispatch]
```

Reply:
But this makes it execute twice for PRs which are pushed to branches on this project.